### ISSUE TITLE
v1.2.1 Thermal Zone/ Space Tab in Large Models

### ISSUE THREAD

**Issue by lgaunt** [openstudiocoalition/OpenStudioApplication#435]
According to the release notes for v1.2.1, the bug that didn't allow the Thermal Zones tab to load in the OS Application was fixed. However, I have a model that is >600 zones (I know, its terrible), that crashes when I open either the space tab or the thermal zone tab.

**Comment by macumber** [comment]
The fix was to increase the stack size on Windows to delay a stack overflow error error for large number of spaces/zones.  Are you using Windows? I made a model with 600 spaces and zones but it does not crash, are you able to share your model?  Also, how much system memory do you have?

**Comment by macumber** [comment]
Thanks for the model, I was able to reproduce the crash.  I will look into it some more.

**Comment by macumber** [comment]
I think the issue is that for large models, the entire grid is painted even when most of the elements are not visible.  This results in a large number of calls to `QWidgetPrivate::paintSiblingsRecursive` and an eventual stack overflow.  

Possible solutions are:

- Refactor to use a Qt item model and table view classes which likely optimize this
- Somehow detect and not draw items that are not visible
- Implement paging or some other way to break up objects so they aren't all in the grid at once

**Comment by macumber** [comment]
I checked and this issue is not resolved when upgrading to Qt 6.2.  Going to defer to a future release as this probably requires a larger refactor.

**Comment by macumber** [comment]
Another instance of this https://unmethours.com/question/69318/openstudio-13-app-crash-using-hvac-zone-tab/

**Comment by macumber** [comment]
@lgaunt I found that the fix for the large model crash was not being applied to the Windows builds on CI.  I just merged a fix for this.  If you have a chance to try the installer below and see if that fixes the issue for you that would be great.  I was able to open your model and use the space and zone tabs.

https://github.com/openstudiocoalition/OpenStudioApplication/suites/5379633885/artifacts/169095988
